docs/installation/autoware/source-installation.md

```diff
@@ -0,0 +1,105 @@
+# Source installation
+
+## Prerequisites
+
+- [Ubuntu 20.04](https://releases.ubuntu.com/20.04/)
+- [Git](https://git-scm.com/)
+  - [Registering SSH keys to GitHub](https://github.com/settings/keys) is preferable.
+
+```bash
+sudo apt-get -y update
+sudo apt-get -y install git
+```
+
+## How to set up a development environment
+
+1. Clone `autowarefoundation/autoware` and move to the directory.
+
+   ```bash
+   git clone https://github.com/autowarefoundation/autoware.git
+   cd autoware
+   ```
+
+2. Install the dependencies.
+
+   We use [Ansible](https://www.ansible.com/) to simplify the steps.
+   Please see the Ansible playbooks and roles if you want to know exactly what packages are installed.
+
+   > Note: Before installing NVIDIA libraries, confirm and agree with the licenses.
+
```

Comment:
```suggestion

> Be very careful with this method. Make sure you read and confirmed all the steps in the ansible configuration before using it.

```

Reply:
I'll use `!!!warning` instead!

Reply:
Actually, this was shown because the branch was old.
But anyway I'll fix this.

Reply:
@xmfcx a061ddea82ff0789e1e96b5504e60155c66b2627
![image](https://user-images.githubusercontent.com/31987104/157906627-90087e0f-5452-4ea7-96ad-9efef665c596.png)

Reply:
Can you pıt the warning before the installation command?

Reply:
Unfortunately, it was impossible to write it in the indented section because of the constraints of `Prettier` and `markdownlint`. :pleading_face: 

![image](https://user-images.githubusercontent.com/31987104/157906897-2cf9bfaa-cb7c-43b9-8402-bbac1a470349.png)

Reply:
Ah ok.

Reply:
Also, it wasn't displayed correctly.
![image](https://user-images.githubusercontent.com/31987104/157907404-90debd4d-6a05-4659-838d-53e370182bb8.png)